elastic/kibana
# Onboard Synthetics TLS rule type with FAAD

Resolves: https://github.com/elastic/kibana/issues/169867

This is the second attempt PR 🙂 to onboard the Synthetics TLS rule type with FAAD.

**To verify**
1. Create an oblt cluster with `/create-ccs-cluster` on slack. Choose `dev-oblt`.
2. Add the configuration values from the oblt command to your kibana.yml. You may have to add:
```
elasticsearch.ignoreVersionMismatch: true
```
and start Kibana
4. Navigate to `app/synthetics/settings/alerting` and add a default connector.
5. Go to `/app/synthetics/monitors/getting-started` and create a HTTP Ping monitor with whatever url you want ( I used https://github.com/) and select a location.
6. Go back to `app/synthetics` and click the Alerts & Rules link. Click TLS certificate rule. Edit the older than param to something low, such as 1 day.
7. The TLS rule should create an active alert, verify that the action message is populated.
8. Repeat step 5 update the older than param to be higher than the age of the cert. You can check your cert here `app/synthetics/certificates`
9. The TLS rule should recover, and verify that the recovery action message is populated.
10. You can also check the AAD docs in dev tools using the following command:
```
GET .internal.alerts-observability.uptime.alerts*/_search
```
